workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -195,4 +199,18 @@
       expect(output_string).to eq(expected_output)
     end
   end
+
+  describe '#buildpr' do
+    pr_number = "pr516"
+    build_id = "CozmoOne_Experiments_NightlyAutomation"
+
+    it 'result contains url build when triggering successfully' do
+      expected_output = "Build has been triggered: #{registry.config.handlers.teamcity.site}/viewQueued.html?itemId=111111"
+      body_str_xml = '<build webUrl="https://build.ankicore.com/viewQueued.html?itemId=111111"></build>'
```

Comment:
url in "expected_output" and in "xml_body_str" is the same, so you can create a common url.